sdcm/coredump.py

```diff
@@ -299,6 +305,34 @@ def n_coredumps(self) -> int:
     def update_coredump_info_with_more_information(self, core_info: CoreDumpInfo):
         pass
 
+    def _get_core_by_pid(self, pid: str) -> Optional[dict]:
+        result = self.node.remoter.sudo(f"coredumpctl list {pid} -q --json=short", verbose=False, ignore_status=True)
+        if not result.ok:
+            return None
+
+        try:
+            cores_info = json.loads(result.stdout)
+        except json.JSONDecodeError:
+            self.log.warning("couldn't parse:\n %s", result.stdout)
+            return None
+        return cores_info[0]
+
+    def _get_executable_version(self, executable: str) -> Optional[str]:
```

Comment:
I was thinking about something generic. To have executable and it's version in a core dump event, in case we would need to do similar skip in future, but for some another service/package

Reply:
let's leave it as is for now